src/config/config.php

```diff
@@ -164,6 +164,12 @@
         // 'my_custom_pagination' => \Path\To\Your\CustomPagination::class,
     ],
 
+    /*
+     * You can define your own pagination type.
+     * Reference \Rebing\GraphQL\Support\PaginationType::class
+     */
+    //'custom_pagination_type' => \Rebing\GraphQL\Support\PaginationType::class,

```

Comment:
I think you should add a `pagination_type` instead that would default to the current `\Rebing\GraphQL\Support\PaginationType::class`

Reply:
Updated